monai/apps/nnunet/nnunetv2_runner.py

```diff
@@ -141,9 +141,12 @@ class nnUNetV2Runner:  # noqa: N801
 
     """
 
-    def __init__(self, input_config: Any, work_dir: str = "work_dir") -> None:
+    def __init__(
+        self, input_config: Any, trainer_class_name: str = "nnUNetTrainer", work_dir: str = "work_dir"
```

Comment:
Please add a doc-string for `trainer_class_name` after L67. The doc-string, in my opnion, should list the available options, e.g. 1epoch, 5epoch and nnUnetTrainer.

Reply:
I have incorporate the definition of `trainer_class_name` into the docstring.